packages/ozone/src/db/schema/setting.ts

```diff
@@ -0,0 +1,24 @@
+import { Generated } from 'kysely'
+import { Member } from './member'
+
+export const settingTableName = 'setting'
+
+export type SettingScope = 'personal' | 'instance'
+
+export interface Setting {
+  id: Generated<number>
+  key: string
+  value: any
```

Comment:
The value here should at minimum be an object due to the way it's modeled as `"type": "unknown"` in lexicon.  (Even if it weren't an object, the `unknown` type may be preferable to `any` here to help with type correctness.)